VERSION

```diff
@@ -1 +1 @@
-2.0.0
\ No newline at end of file
+2.1.1
```

Comment:
Why 2.1 ? any breaking change? Or just added features to 2.0.0?

Reply:
I considered replacing `ensembl_name` with `production_name` as a breaking change -> `2.1.0`
and removing `ensembl_name` from `GetOrganismsGroupCount()` endpoint as a minor change -> `2.1.1`
I hope this is how it works

Reply:
Well, I agree, in principle yes, just that 2.0.0 is still in dev, so it's already a big breaking change, and is not released yet. So it's simpler to keep with 2.0.0 (major break) including all those changes? And the VERSION is updated only when previous ones is released? Does that make sense?

Reply:
I see, that makes sense, I reversed it back to 2.0.0